### PR TITLE
Issue #6966: aligned javadoc/xdoc for AvoidEscapedUnicodeCharacters

### PR DESCRIPTION
**Pull request by pbludov**
Issue #6966 (part of #5750)

No changes in the code, but many changes in the comments.

### CODE REVIEW

File: src/xdocs/config_misc.xml [hunk 1]
```diff
@@ -271,7 +271,8 @@ String unitAbbrev = "\u03bc\u03bc\u03bc";
 &lt;/module&gt;
         </source>
         <p>
-          An example of how to configure the check to allow non-printable escapes:
+          An example of how to configure the check to allow using escapes
+          for non-printable whitespace characters:
```
Comment (rnveach) [inline src/xdocs/config_misc.xml]: `non-printable whitespace` => `non-printable, whitespace`

File: src/xdocs/config_misc.xml [hunk 2]
```diff
@@ -226,14 +226,14 @@ String unitAbbrev = "\u03bcs"; //Poor: the reader has no idea what this is.
...
-          An example of non-printable(control) characters.
+          An example of non-printable control characters.
         </p>
         <source>
 return '\ufeff' + content; // byte order mark
         </source>
         <p>
           An example of how to configure the check to allow using escapes
-          for non-printable(control) characters:
+          for non-printable control characters:
```
Comment (rnveach) [inline src/xdocs/config_misc.xml]: `non-printable control` => `non-printable, control`

File: src/xdocs/config_misc.xml [hunk 3]
```diff
@@ -226,14 +226,14 @@ String unitAbbrev = "\u03bcs"; //Poor: the reader has no idea what this is.
 &lt;module name=&quot;AvoidEscapedUnicodeCharacters&quot;/&gt;
         </source>
         <p>
-          An example of non-printable(control) characters.
+          An example of non-printable control characters.
```
Comment (rnveach) [inline src/xdocs/config_misc.xml]: same

File: src/xdocs/config_misc.xml [hunk 4]
```diff
@@ -182,30 +182,30 @@ public class MyClass {
...
             <td>Allow if all characters in literal are escaped.</td>
             <td><a href="property_types.html#boolean">Boolean</a></td>
-            <td>false</td>
+            <td><code>false</code></td>
             <td>5.8</td>
           </tr>
           <tr>
             <td>allowNonPrintableEscapes</td>
-            <td>Allow non-printable escapes.</td>
+            <td>Allow use escapes for non-printable whitespace characters.</td>
```
Comment (rnveach) [inline src/xdocs/config_misc.xml]: same

File: src/xdocs/config_misc.xml [hunk 5]
```diff
@@ -182,30 +182,30 @@ public class MyClass {
           </tr>
           <tr>
             <td>allowEscapesForControlCharacters</td>
-            <td>Allow use escapes for non-printable(control) characters.</td>
+            <td>Allow use escapes for non-printable control characters.</td>
```
Comment (rnveach) [inline src/xdocs/config_misc.xml]: same

File: src/xdocs/config_misc.xml (hunk 1 above)
Comment (pbludov) [reply src/xdocs/config_misc.xml]: done. Fixed it wherever I found it.

**Pull request merged**